src/reducers/transactionBuilder.js

```diff
@@ -106,6 +109,10 @@ function attributes(state = defaultAttributes, action) {
     return Object.assign({}, state, action.newAttributes);
   case FETCH_SEQUENCE_SUCCESS:
     return Object.assign({}, state, { sequence: action.sequence });
+  case FETCH_BASE_FEE_SUCCESS:
+    return Object.assign({}, state, {fee: action.base_fee});
```

Comment:
Small, nonblocking detail: by looking at the [lines above](https://github.com/stellar/laboratory/pull/460/files#diff-afb08b5ed40971376e57c51329828e12R110-R111) it seems like we use were adding a whitespace between the curly braces and the content – I think we should keep it for standardization purposes. Maybe it's just a matter of running `prettier`?